refactor/src/Cage.py

```diff
@@ -418,13 +426,36 @@ def do_mating_events(self):
                 df_list[fc].loc[df_list[fc].index.isin(dams),'mate_resistanceT1'] = \
                 df_list[fc].loc[df_list[fc].index.isin(sires),'resistanceT1'].values
         """
-        pass
 
-    def create_offspring(self):
+        # for now only consider a sensible number of lice that can mate
+        return min([10, self.lice_population["L5f"], self.lice_population["L5m"]])
+
+    def get_num_eggs(self, cur_month) -> int:
         """
-        TODO
+        Get the number of new eggs
+        :param cur_month the current month (to compute the temperature)
+        :returns the number of eggs produced
         """
 
+        # See Aldrin et al. 2017, §2.2.6
+        # TODO: in Aldrin matings are not taken into account. This may change
+        female_population = self.lice_population['L5f']
```

Comment:
Can we keep consistency in usage of `'` and `"`?

Reply:
Note: it would be good to adopt a linting strategy that covers this and other similar situations. See #14